tests/BinaryXml/SoapDecoderTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace CasperBiering\Dotnet\Tests\BinaryXml;
+
+use CasperBiering\Dotnet\BinaryXml\SoapDecoder;
+
+class SoapDecoderTest extends \PHPUnit_Framework_TestCase
+{
+    public function testSample()
+    {
+        $binary = file_get_contents(__DIR__.'/soap1.bin');
+        $expected = file_get_contents(__DIR__.'/soap1.xml');
+        $decoder = new SoapDecoder();
+
+        $actual = $decoder->decode($binary);
+
+        $this->assertXmlStringEqualsXmlString($actual, $expected);
```

Comment:
Good catch! I just submitted a fix to this branch.
Please let me know if you spot any further issues.